packages/material-ui/src/Popover/Popover.js

```diff
@@ -110,16 +110,14 @@ class Popover extends React.Component {
   };
 
   setPositioningStyles = element => {
-    if (element && element.style) {
```

Comment:
I believe these are defensive checks that were added during the Flow coverage of the library.
Most of them are useless or hide core issues. Let's try without!